federated_aws_cli/role_picker.py

```diff
@@ -38,6 +39,27 @@ def get_aws_env_variables(credentials):
     return result
 
 
+def get_aws_shared_credentials(credentials, role_arn):
+    # come up with a profile name that won't break AWS
+    result = "[{}]\naws_access_key_id={}\naws_secret_access_key={}\nrole_arn={}\n".format(
+        re.sub("[^a-zA-Z0-9]", "", role_arn),
+        credentials.get("AccessKeyId", ""),
+        credentials.get("SecretAccessKey", ""),
+        role_arn
+    )
+    verb = "set" if platform.system() == "Windows" else "export"
+    path = write_aws_shared_credentials(role_arn, result)
+
+    if path:
+        return "{} AWS_SHARED_CREDENTIALS_FILE={}\n" \
+               "{} AWS_SESSION_TOKEN={}".format(
```

Comment:
This can be written into the AWS_SHARED_CREDENTIALS_FILE file. The `aws_session_token` variable can contain this

Reply:
Oh, good to know!